samples/quickstart.js

```diff
@@ -17,24 +17,51 @@
 // sample-metadata:
 //   title: Quickstart
 //   description: A quick introduction to using the Pub/Sub client library.
-//   usage: node quickstart.js <project-id> <topic-name>
+//   usage: node quickstart.js <project-id> <topic-name> <subscription-name>
+
+function exit(code) {
+  // eslint-disable-next-line no-process-exit
+  process.exit(code);
```

Comment:
Having a function that calls a single line of code feels weird.  If this is to avoid copying the eslint ignore rule, I'd suggest upleveling it to `samples/.eslintrc.json` as a custom rule.

Reply:
I feel pretty weird about this too, for a number of reasons. I'm good with making it a linter rule change, I just don't like to presume that stuff :)